Android--EN/domain/BroadcastState.java

```diff
@@ -1,7 +1,7 @@
 package com.herewhite.sdk.domain;
 
 /**
- * The view state of the local user, as well as the user information of the host (if any) in the room.
+ * The view state of the local user, as well as the user information of the host (if any) in the room. //TODO William Is there a difference between view "state" and view "mode"? Should this be "mode"?
```

Comment:
The view state is mainly about the view mode of the local user, but also includes information on broadcasterId and broadcasterInformation.